apps/client/src/module/auth/user-store.ts

```diff
@@ -36,6 +37,7 @@ export const fetchUserInfo = async () => {
     title: "Software Engineer",
```

Comment:
I'm thinking, since now all user info are retrieved from back-end, we need to remove those fake data.

Please clean them, also check the profile page to make sure it's not too strange after removing those

Reply:
deleted fake data